app/assets/stylesheets/scaffolds.scss

```diff
@@ -1,12 +1,44 @@
 body {
   background-color: #fff;
   color: #333;
-  margin: 33px; }
+  margin: 33px; 
+  margin-top: 100px;
+}
 
 body, p, ol, ul, td {
   font-family: verdana, arial, helvetica, sans-serif;
-  font-size: 13px;
-  line-height: 18px; }
+  font-size: 21px;
+  line-height: 26px; }
+
+header {
```

Comment:
It would be nice to move these header, logo, and nav styles to the layouts stylesheet, since they correspond to elements that are defined in `layouts/application.html.erb`